etsd/urls.py

```diff
@@ -15,12 +15,13 @@
     # path("users/", include("etsd.users.urls")),
     path("messages/", include("etsd.msgs.urls")),
     path("keys/", include("etsd.keys.urls")),
+    path("users/", include("etsd.users.urls")),
     path("admin/", admin.site.urls),
     path('authorities/', permission_required('authorities.view_authority')(AuthorityListView.as_view()), name='authority_list', ),
     path('authorities/create', permission_required('authorities.add_authority')(AuthorityCreateView.as_view()), name='authority_create', ),
```

Comment:
Change these permission_required to `core_admin`. So. For create/view/update the required permission will be core.admin. For edit users it will by core.admin or core.user. When a core.user logs in he'll see in the /home a button that will open the edit users view if *his* authoiry. When a core.admin user logs in he'll see a button to go to the authority list view.